modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/BasePhenotypeAnnotation.pm

```diff
@@ -376,6 +376,36 @@ sub default_phenotype_class {
 #----------------------------
 # PUBLIC METHODS
 
+=head2 update_meta_coord
+
+  Example    : $obj->update_meta_coord()
+  Description: Update phenotype_feature meta_coord entries in variation db
+  Returntype : none
+  Exceptions : none
+
+=cut
+
+sub update_meta_coord {
+  my $self = shift;
+
+  my $core_db_name = $self->core_db_adaptor->dbc()->dbname();
+  my $vdbh =  $self->variation_db_adaptor->dbc();
+  my $table_name ='phenotype_feature';
+
+  # delete previous phenotype_feature entries in update_meta_coord
+  my $mc_del_sth = $vdbh->do(qq[ DELETE FROM meta_coord WHERE table_name ='$table_name' ]);
```

Comment:
The values returned by the do statements are not used. Variable suggests that it is a  statement handler.  do returns number of rows  or undef on error.